scripts/user_functions.py

```diff
@@ -185,6 +156,35 @@ def eval_expr(expr: str) -> str:
     return f"{eval(expr)}"
 
 
+def extract_keywords_from(parts: List[str]) -> Dict[str, str]:
+    """
+    Given a list of strings, extract strings containing an equal sign ("=").
+
+    Return a *defaultdict(str)* with key=value extracted from the original list.
+
+    The left part of the sign is used a as the dict key and the right part as the value value.
+    When a string contains the sign, it is removed from the original list.
```

Comment:
FWIW, I found one occurence when this cause an issue: https://en.wiktionary.org/wiki/EPR=ER

Reply:
Hm this is a tricky one :)